Makefile

```diff
@@ -41,7 +41,7 @@ endif
 
 ifneq "" "$(shell which cargo)"
 optional-rust-support:
-	cd rust && cargo build --release
+	cd rust && RUSTFLAGS="-C target-cpu=native" cargo build --release
```

Comment:
Ow, I thought that `make` is run directly on the server and that's how the service is running.
I'll fix that(I did that to be able to enable simd optimizations).